payments/src/main/java/it/gov/pagopa/payments/service/PaymentsService.java

```diff
@@ -176,53 +227,27 @@ private PaymentsResult<ReceiptEntity> getSegmentedReceipts(Integer limit, Intege
 
 	private PaymentsResult<ReceiptEntity> getNotSegmentedReceipts(CloudTable table, String filter) {
 		 final int FIRST_PAGE_NUMBER = 0;
-		 String[] columns = new String[]{PARTITION_KEY_FIELD, ROW_KEY_FIELD, DEBTOR_FIELD};
+		 String[] columns = new String[]{PARTITION_KEY_FIELD, ROW_KEY_FIELD, DEBTOR_FIELD, STATUS_FIELD};
```

Comment:
why not a class `private static final String[] columns` both for segmentation or not. ?

Reply:
Yes, you are right, it is a useless duplication